lib/element.ts

```diff
@@ -447,6 +446,14 @@ export class ElementArrayFinder extends WebdriverWebElement {
     });
   }
 
+  getText(): wdpromise.Promise<any> {
```

Comment:
I had to return here "any" because in the interface it's defined as string, but string[] would be needed here.
It is also not documented that ElementArrayFinder getText() is available. But there are tests where getText is used on ElementArrayFinder.

Reply:
It doesn't quite make sense for me to have getText() for ElementArrayFinder
As you said, there are tests where getText is used on ElementArrayFinder, what are the tests that you mentioned?

Reply:
@qiyigg 

I must say that I use `getText()` a lot on `ElementArrayFinder`, it is possible, so a fix on the typings would be great.
What @viktorpergjoka means is also the amount of examples in this files that `getText()` is used on `ElementArrayFinder`'s

Reply:
Regarding the tests for example in elements_spec.js:
describe('ElementArrayFinder', function() {

  it('action should act on all elements', function() {
    browser.get('index.html#/conflict');

    var multiElement = element.all(by.binding('item.reusedBinding'));
    expect(multiElement.getText()).toEqual(['Outer: outer', 'Inner: inner']);
  });

Reply:
oh, I see. The function name is a little misleading.